src/plugins/intel_cpu/src/dnnl_postops_composer.h

```diff
@@ -95,6 +97,7 @@ class DnnlPostOpsComposer {
     const dnnl::memory::data_type outDataType;
     bool useLegacyPostOps;
     bool useLegacyZeroPoints;
+    bool forceLegacyPostOps;
```

Comment:
Should it be a const?

Reply:
Ideally, they all should not be const, but let's make it const for now.
We have clang-tidy check which should resolve this